sources/tests/Fixtures/PikaChuJuicer.php

```diff
@@ -38,12 +39,7 @@ protected function doTrimAndLowerString($value): string
     protected function mustNotBeEmptyString($value)
     {
         if (strlen($value) === 0) {
-            throw new ValidationException(
-                sprintf(
-                    "Field '%s' is an empty string.",
-                    $name
-                )
-            );
+            throw new ValidationException("Field 'pika' is an empty string.");
```

Comment:
ah ah no, there is no need to handle the field name in the filter / validator. It is handled in the exception in the upper layer. Just a string `cannot be empty` is enough.